test/unit/lib/plugins/aws/package/compile/events/iotFleetProvisioning/index.test.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../../../utils/run-serverless');
+const templateBody = require('../../../../../../../../fixtures/iotFleetProvisioning/template.json');
+
+chai.use(require('chai-as-promised'));
+
+const { expect } = chai;
+
+describe('AwsCompileIoTFleetProvisioningEvents', () => {
```

Comment:
Let's title test after file name so `lib/plugins/aws/package/compile/events/iotFleetProvisioning/index.test.js ` (sorry, it's a new convention, we're incorporating now in new and refactored tests)